### ISSUE TITLE
Replication without blobs?

### ISSUE THREAD

**Issue by krissik** [zopefoundation/zc.zrs#8]
Our blob files are saved on a well-backuped NAS and shared between plone workers.
Now we consider using ZRS and are wondering if we have to replicate the blobs (~190 GB) or can they be shared between primary and secondary too?

I have tested this buildout for primary
[primary_buildout.txt](https://github.com/zopefoundation/zc.zrs/files/4993466/primary_buildout.txt)
and this for secondary
[secondary_buildout.txt](https://github.com/zopefoundation/zc.zrs/files/4993468/secondary_buildout.txt)

"blob-storage" in this test is the *same* local directory. It seems to work but if I 
* deactivate the secodary, 
* create an image in plone, 
* check the creation date of the corresponding blob file, 
* activate secondary and 
* check creation date again 

I see that the creation date is the time of activating secondary. So I come to the conclusion that the blob file is replicated and saved again to its original location after activating secondary.

In short: Is there a way to deactivate replication of blob files and share them between primary and secondary?

**Comment by krissik** [comment]
In case we want to implement an option to control if blobs are copied from primary to secondary what do we need to consider? Are there any pitfalls we have to be aware of?
